Change passed arguments to transitions

ar4s opened the pull request:
Main changes:
 - pass ``requester`` argument to precondition function
 - refactor - pass ``requester`` argument instead of ``request`` in transition function

mkurek commented on src/ralph/lib/transitions/async.py:
```diff
@@ -60,9 +60,14 @@ def run_async_transition(job_id):
     transition = transition_job.transition
     obj = transition_job.obj
-    _check_instances_for_transition([obj], transition, check_async_job=False)
+    requester = transition.jobs.first().user
```
why don't you use `transition_job` object?

mkurek commented on src/ralph/lib/transitions/models.py:
```diff
@@ -424,7 +436,7 @@ def run_field_transition(
         _post_transition_instance_processing(
             instance, transition, data, history_kwargs=history_kwargs,
-            user=kwargs['request'].user, attachments=attachments,
+            user=requester, attachments=attachments,
```
maybe `user` param could be renamed to `requester` as well to keep it consistent?

mkurek reviewed (commented):
So the main question here is, why (do you want to change it)? :)

Commit:
Change passed arguments to transitions

Main changes:
 - pass ``requester`` argument to precondition function
 - refactor - pass ``requester`` argument instead of ``request`` in transition
   function

Commit:
Merge branch 'ng' into transitions/requester-instead-of-request
